.travis.yml

```diff
@@ -1,20 +1,24 @@
+dist: xenial
 language: rust
 rust:
   - nightly
   - beta
-  - 1.28.0 # stable
+  - 1.31.0 # stable
 env:
-  - GTK=3.4
-  - GTK=3.18
+  - GTK=3.14
+  - GTK=3.24
 addons:
   apt:
     packages:
     - libgtk-3-dev
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ]; then
     make regen_check;
     fi
+  - for crate in "atk-sys gdk-pixbuf-sys gdk-sys gio-sys glib-sys gobject-sys gtk-sys pangocairo-sys pango-sys"; do
+    cd $crate && cargo test && cd ..
```

Comment:
Better don't do this now, it take too much time and fails.

Reply:
Ah so nobody actually fixed the test since they were merged... disappointing :( Then I disable that again.